charts/generic/pod-http-reset-peer/pod-http-reset-peer.chartserviceversion.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: litmuchaos.io/v1alpha1
+kind: ChartServiceVersion
+metadata:
+  name: pod-http-reset-peer 
+  version: 0.1.0
+  annotations:
+    categories: generic
+spec:
+  displayName: pod-http-reset-peer 
+  categoryDescription: >
+    it injects http reset into the pod which stops outgoing http requests by resetting the TCP connection and then reverts back to the original state after the specified duration 
+  keywords: 
+    - "pods" 
+    - "kubernetes" 
+    - "generic" 
+    - "exec" 
+    - "http" 
+    - "loss"
+  platforms: 
+    - "Minikube"
+  maturity: alpha
+  maintainers: 
+    - name: Akash Shrivastava 
+      email: akash.shrivastava@harness.io 
+  minKubeVersion: 1.12.0 
+  provider: 
+    name: Harness
+  labels:
+    app.kubernetes.io/component: chartserviceversion
+    app.kubernetes.io/version: latest
+  links: 
+    - name: Documentation 
+      url: https://docs.litmuschaos.io/docs/getstarted/ 
```

Comment:
This is also a common task across many other experiment, I will create an issue for it.

Reply:
I think all the experiments have the Documentation and Source Code fields.